packages/beasties/src/index.ts

```diff
@@ -92,13 +90,12 @@ export default class Beasties {
 
     // go through all the style tags in the document and reduce them to only critical CSS
     const styles = this.getAffectedStyleTags(document)
-
```

Comment:
is there a reason we need to do them serially rather than in parallel?

Reply:
Since there is no asynchronous operation under the hood they are still done serially since Node.js is single threaded and these operations are not I/O bound.